components/automate-ui/src/app/pages/team-details/team-details.component.ts

```diff
@@ -65,7 +65,7 @@ export class TeamDetailsComponent implements OnInit, OnDestroy {
   ) {
     this.updateNameForm = fb.group({
       // Must stay in sync with error checks in team-details.component.html
-      name: ['Loading...', [Validators.required, Validators.pattern(Regex.patterns.NON_BLANK)]]
+      name: ['', [Validators.required, Validators.pattern(Regex.patterns.NON_BLANK)]]
```

Comment:
The functionality needs to have no initial value specified upon form creation. I don't see loss of this brief flash of `Loading...` as causing any significant user degradation.